### DISCUSSION TITLE
SofaPython3 not working

### DISCUSSION THREAD

**Discussion by oataga** [sofa-framework/sofa#3867] [Build / Config / Environment]
As in the title, I cannot overcome the problem of a plugin that does not load correctly. 
I installed sofa from their website, throught the .exe file (I work on Windows).
And once I try to launch a .py simulation, or basically any simulation with Python extension, I get the following error:

[ERROR]   [DAGSimulation(Simulation)] Cannot load file 'C:\Users\******\Desktop\SOFA\main1.pyscn': extension (pyscn) is only supported if the plugin SofaPython3 is loaded. SofaPython3 must be loaded first before being able to load the file.

Here I'm stuck, I have no idea how to fix this or what steps should I take.

**Comment by alxbilger** [comment]
As the error message suggests, you need to load SofaPython3 in order to be able to load a Python scene. See the documentation: https://www.sofa-framework.org/community/doc/plugins/what-is-a-plugin/#plugin-loading

**Reply by oataga** [reply]
I did that, got a following message: 
[ERROR]   [PluginManager] Plugin loading failed (C:/Users/Oataga/SOFA/v22.12.00/bin/SofaPython3.dll): Nie mo|na odnalez okre[lonego moduBu.

that message after ....dll): means in polish **_"Couldn't find a given module"._**

**Reply by oataga** [reply]
Nevermind, I got that to work. Thanks for your help!!!

**Reply by weiweihaoshuai** [reply]
How did you solve it? Can you help me?
